source/layers/validation/ze_valddi.cpp

```diff
@@ -13,6 +13,15 @@
 
 namespace validation_layer
 {
+    // Forward declaration for Intel experimental extension
```

Comment:
This file is generated thru https://github.com/oneapi-src/level-zero/blob/master/scripts/templates/validation/valddi.cpp.mako . You need to modify the .mako file to add this functionality, otherwise at the spec update your code will be removed.